acl/src/test/java/org/apache/rocketmq/acl/plain/PlainPermissionManagerTest.java

```diff
@@ -32,7 +32,9 @@
 import org.assertj.core.util.Lists;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
```

Comment:
This seems to be the imports of another pull request.

Reply:
Should we think about merging #4637 and then #4991